### DISCUSSION TITLE
Customize name id __next

### DISCUSSION THREAD

**Discussion by pablosantos22** [vercel/next.js#48387] [Help]
### Summary

Hello!
At the company I work for, we use micro-frontends and we need to create a preload where the static files would be ready to be rendered inside a div with an id related to the micro-frontend so that there is no collision. I haven't found a way to customize the id of this root div. The idea would be to have id=app_1, id=app_2 etc..

**Comment by devtombiz** [comment]
I would be interested to know how to do that too ! 
I am trying to integrate an added component in a client app already using next.js so there are conflicts between the client app "__next" div and my app as it tries to override the client app. It would be great to be able to scope the "__next" div and style.
So far I have found this article: https://www.codingcrossroads.com/decorating-the-next-js-root-div/ but I haven't been able to make it work so far.
